pkg/git/fetch.go

```diff
@@ -18,56 +28,84 @@ type Fetcher struct {
 func (f Fetcher) Fetch(dir, gitURL, gitRevision, metadataDir string) error {
 	f.Logger.Printf("Cloning %q @ %q...", gitURL, gitRevision)
```

Comment:
Do we need to create a tempdir (https://github.com/pivotal/kpack/blob/d4de54b914365f6082bcbe938fa038aab18ed9b9/pkg/git/fetch.go#L30)?

Reply:
https://github.com/pivotal/kpack/blob/d4de54b914365f6082bcbe938fa038aab18ed9b9/pkg/git/fetch.go#L76

Reply:
Yes, in order to preserve permissions on copy